File: Laboratórios/Empresa e Departamentos/Pessoa.java
import java.time.LocalDate;

public class Pessoa{
    private String nome;
    private LocalDate nascimento;

    public Pessoa(String nome, int year, int month, int day){
        this.nome = nome;
        this.nascimento = LocalDate.of(year, month, day);
    }

    public LocalDate getNascimento() {
        return nascimento;
    }

    public String getNome() {
        return nome;
    }

    public int getIdade(){
        LocalDate atual = LocalDate.now();

        int idade = atual.getYear() - this.nascimento.getYear();

        if(atual.getMonthValue() < this.nascimento.getMonthValue()){
            idade--;
        }else if(atual.getMonthValue() == this.nascimento.getMonthValue()){
            if(atual.getDayOfMonth() < this.nascimento.getDayOfMonth()){
                idade--;
            }
        }

        return idade;
    }
}

File: Aulas/C++/Aula 1/pessoa.cpp
#include "pessoa.hpp"

#include <iostream>

using namespace std;

Pessoa::Pessoa(const string &nome, int idade) : nome(nome), idade(idade)
{
    cout << "Construímos uma pessoa de nome " << nome << " e idade " << idade << endl;
};

void Pessoa::setNome(const string &nome)
{
    this->nome = nome;
}

string Pessoa::getNome() const
{
    return this->nome;
}

int Pessoa::getIdade() const
{
    return this->idade;
}
